core/src/main/java/io/grpc/transport/AbstractStream.java

```diff
@@ -348,7 +348,7 @@ protected final void onSendingBytes(int numBytes) {
    *
    * @param numBytes the number of bytes that were sent.
    */
-  protected final void onSentBytes(int numBytes) {
+  public void onSentBytes(int numBytes) {
```

Comment:
I still liked the idea that this would be overridden in Netty to be public but would otherwise be protected.